src/apps/frontend/hooks/login-form.hook.ts

```diff
@@ -0,0 +1,50 @@
+import { useFormik } from 'formik';
+import { useState, useCallback } from 'react';
+import * as Yup from 'yup';
+
+import constant from '../constants';
+import { AccessService } from '../services';
+
+const useLoginForm = () => {
+  const [success, setSuccess] = useState(false);
+  const [error, setError] = useState(false);
+
+  const login = useCallback(
+    async ({ username, password }) => {
+      setSuccess(false);
+      setError(false);
+
+      try {
+        const accessService = new AccessService();
```

Comment:
You can move the initialization of this outside this UI component i.e. 

```
const accessService = new AccessService();
const useLoginForm = () => {
```

Should we also move this closer to the login form, and avoid creating another folder `hooks`? I can see in future everyone would put all context here which would make this folder explode

also, `AccessService` seems mislead, recommend to rename to better reflect what it is.

Reply:
Done, renamed it to AuthService